release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -503,6 +503,17 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-17-etcd-certificates_{context}"]
 === Security
 
+[id="ocp-17-auto-rotate-etcd-ca"]
```

Comment:
```suggestion
[id="ocp-17-auto-rotate-etcd-ca_{context}"]
```

Example from the 4.16 release notes: 
![Screenshot from 2024-08-30 10-51-04](https://github.com/user-attachments/assets/3b9ecb0d-7d20-48fa-86f2-aff441333c6a)